app/encoding.go

```diff
@@ -2,15 +2,40 @@ package althea
 
 import (
 	"github.com/althea-net/althea-chain/app/params"
-	"github.com/cosmos/cosmos-sdk/std"
+
+	ethermintenccdc "github.com/evmos/ethermint/encoding/codec"
 )
 
```

Comment:
The registrations that happen here are confusing so I tracked down each registration to compare with the standard sdk encoding registrations. Ultimately the Ethermint registrations call all the standard registrations, but now we know everything else they are doing via the comments.

Reply:
really great work here documenting this.